src/main/python/deep_qa/training/trainer.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, params: Dict[str, Any]):
         # Upper limit on the number of training instances.  If this is set, and we get more than
         # this, we will truncate the data.
         self.max_training_instances = params.pop('max_training_instances', None)
+        self.max_tuning_instances = params.pop('max_tuning_instances', None)
```

Comment:
I think it makes more sense to just change the API so that `train_files` is optionally a list of lists, so you can specify some number of datasets to train the model with, sequentially. Then you don't have to add another parameter if you want to train on three datasets in turn, and everything is just cleaner.